.editorconfig

```diff
@@ -11,9 +11,11 @@
 indent_size = 2
 ij_kotlin_allow_trailing_comma_on_call_site = true
 ij_kotlin_allow_trailing_comma = true
+ktlint_code_style = intellij_idea
```

Comment:
This file mentions it is auto generated and any changes to it will be overridden on your next build. 
And to make changes to this file it asks to change the gradle plugin dps-gradle-spring-boot,
publish a new version and update to the new version in your gradle build script.
Is this not the case?

Reply:
Indeed! However, I haven't changed this file myself. I assume it was re-auto-generated in the build.